src/components/custom-field-input-number/custom-field-input-number.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import CustomFieldInputText from '../custom-field-input-text/custom-field-input-text.jsx';
+import CustomFieldInputNumber from './custom-field-input-number.jsx';
+
+describe('CustomFieldInputNumber', () => {
+  it('has defaults', () => {
+    const tree = renderer.create((
+      <CustomFieldInputNumber />
+    )).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  describe('prop-forward API', () => {
+    it('forwards all props accepted by CustomFieldInputText', () => {
```

Comment:
Looking over this test a second time... I'm not entirely confident it covers the intention that the props are forwarded. 

I think for _us_, it makes sense that the props on each interface. But if we fast-forward a year or two into the future, a new engineer might not know that this is suppose to cover the intention of forwarding the props. 

I am not sure if is possible to have a simple test for this -- especially with `@testing-library/react`. Usually, we can just expect the props from `CustomFieldInputNumber` to be on the instance of `CustomFieldInputText ` but the testing library wants tests written from the perspective of user interactions. We might want to use `react-test-renderer` to see the props being forwarded?